src/monotonic/monotonic.jl

```diff
@@ -189,13 +194,27 @@ function derivative(itp::MonotonicInterpolation, x::Number)
     return itp.m[k] + 2*itp.c[k]*xdiff + 3*itp.d[k]*xdiff*xdiff
 end
 
+function hessian(itp::MonotonicInterpolation, x::AbstractArray{<:Number, 1})
+    length(x) == 1 || error("Given vector x ($x) should have exactly one element")
```

Comment:
Same thing here: `x` should be `VarArg`, and the length check is then probably redundant.